### PR TITLE
Update maintainers for Juju charm layers

### PR DESCRIPTION
**Pull request by ktsakalozos**
**What this PR does / why we need it**: Update maintainers of harm layers to reflect ... reality

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 

**Special notes for your reviewer**:

**Release note**:

```NONE
```

### CODE REVIEW

File: cluster/juju/layers/kubeapi-load-balancer/metadata.yaml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 name: kubeapi-load-balancer
 summary: Nginx Load Balancer
 maintainers:
-  - Charles Butler <charles.butler@canonical.com>
```
Comment (chuckbutler) [inline cluster/juju/layers/kubeapi-load-balancer/metadata.yaml]: 😢

Comment (Cynerva) [inline cluster/juju/layers/kubeapi-load-balancer/metadata.yaml]: @chuckbutler We can leave you on the maintainer lists if you want, we're mostly concerned about the non-existent email address. What would you prefer?

**Pull request merged**